buidlerConfigSol6.js

```diff
@@ -135,7 +135,8 @@ module.exports = {
             "0x28d1bfbbafe9d1d4f5a11c3c16ab6bf9084de48d99fbac4058bdfa3c80b2908f",
           balance: "100000000000000000000000000000000"
         }
-      ]
+      ],
+      allowUnlimitedContractSize: true
```

Comment:
and if we need it.
is it required both sol4 and sol6?

Reply:
Only sol6, cos it's only the malicious network contracts that exceed.